test.js

```diff
@@ -7,20 +8,25 @@ it('should return the name of named functions', function () {
 });
 
 it('anonymous', function () {
+	/* eslint-disable */
```

Comment:
To test a function without space before parenthesis.

```js
assert.equal(fnName(function() {}), null);
```

It produces an error.
```js
  11:30  error  Missing space before function parentheses  space-before-function-paren
```

Do you have a better idea?